task/buildah/0.1/buildah.yaml

```diff
@@ -543,16 +543,10 @@ spec:
     workingDir: $(workspaces.source.path)
 
   - name: upload-sbom
-    image: quay.io/redhat-appstudio/cosign:v2.1.1@sha256:c883d6f8d39148f2cea71bff4622d196d89df3e510f36c140c097b932f0dd5d5
-    args:
-      - attach
-      - sbom
-      - --sbom
-      - sbom-cyclonedx.json
-      - --type
-      - cyclonedx
-      - $(params.IMAGE)
+    image: quay.io/konflux-ci/appstudio-utils:ab6b0b8e40e440158e7288c73aff1cf83a2cc8a9@sha256:24179f0efd06c65d16868c2d7eb82573cce8e43533de6cea14fec3b7446e0b14
     workingDir: $(workspaces.source.path)
+    script: |
+      cosign attach sbom --sbom sbom-cyclonedx.json --type cyclonedx "$(cat "$(results.IMAGE_REF.path)")"
```

Comment:
I think that the 0.1 version of `buildah*` doesn't have `IMAGE_REF`. You could probably use `IMAGE_URL` and `IMAGE_DIGEST` instead (or just keep using `params.IMAGE` like it was before)

Reply:
do we need to update the 0.1 versions? Recently @mmorhun brought up that we would like to deprecate them and them remove them in time anyway.

Reply:
Probably not. The presence of the quay.io/redhat-appstudio/cosign image in the repo may raise some questions, but a few rounds of justifying it to people should do the trick (if that's easier than making the change)

Reply:
Let's not change stuff we are gonna remove soon